src/config/index.ts

```diff
@@ -24,6 +24,8 @@ export const GOCD_ORIGIN =
   process.env.GOCD_ORIGIN || 'https://deploy.getsentry.net';
 export const FEED_DEPLOY_CHANNEL_ID =
   process.env.FEED_DEPLOY_CHANNEL_ID || 'C051ED5GLN4';
+export const FEED_OPTIONS_AUTOMATOR_CHANNEL_ID =
+  process.env.FEED_OPTIONS_AUTOMATOR_CHANNEL_ID || 'C05JJ5JNZAB';
```

Comment:
Where did you pick this channel id?
#proj-options-automator seems to be `C04URUC21C5`

Reply:
this was my test channel id while I was testing locally, will correct it before merging.